server/welcomebot.go

```diff
@@ -45,6 +45,29 @@ func (p *Plugin) constructMessageTemplate(userID, teamID string) *MessageTemplat
 	return data
 }
 
+func (p *Plugin) getGlobalMessageTemplateData(userID string) *GloablMessageTemplate {
+	data := &GloablMessageTemplate{}
+	var err *model.AppError
```

Comment:
`appErr` is usually used for this to make this obvious

```suggestion
	var appErr *model.AppError
```